homeassistant/components/syncthru/strings.json

```
@@ -0,0 +1,27 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "This device has already been configured"
```

Please use references. 
```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```